packages/client/src/components/text-logo.svg

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This should be moved to assets and cleaned up

https://fvsch.com/svg-icons

Reply:
Text-logo was using paths with numerous anchors for the text portion, so many that when cleaned/simplified the text became really distorted.  Latest commit replaces with a version using text in the svg file instead of paths

Reply:
Just realized that the text-logo font style and color can now be moved out of the svg to NavStyle, and controlled by theme :)